Makefile

```diff
@@ -1,12 +1,18 @@
-.PHONY: clean build-image
+.PHONY: clean check_prep check
 
-#   clean: clean up all build artifacts.
-#   build-image: build docker image for selected projects.
+DOCKER ?= $(shell which docker)
+IMAGE_NAME="goovn:test"
 
-# Remove all build artifacts.
-#
-# Example:
-#   make clean
+.PHONY=check
+check: check_prep
+	$(DOCKER) run -e "OVN_SRCDIR=/src/" -v $$PWD:/root/workspace -w /root/workspace -it $(IMAGE_NAME) .travis/test_run.sh
```

Comment:
For the docker approach, "go get -v" is executed every time, but the result is lost after container exit. This step is also time-consuming. So I wonder if it is better to have it in test_prepare.sh as well. That way, the subsequent "make check" would be much faster.

Reply:
Maybe we run 'go get' in both scripts? That way live modifications of the go.mod file will be picked up while the go mod cache speeds things up. WDYT?

Reply:
Yes, sounds good.

Reply:
Done